hadoop-hdds/docs/themes/ozonedoc/static/swagger-resources/recon-api.yaml

```diff
@@ -0,0 +1,1584 @@
+openapi: 3.0.0
+info:
+  title: Ozone Recon REST API
+  license:
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+    name: Apache 2.0 License
+servers:
+  - url: /api/v1/
+tags:
+  - name: Containers and Keys
+    description: APIs to fetch information about the available containers and keys. **Admin Only**
+  - name: Blocks Metadata
+    description: APIs to fetch metadata for the blocks available. **Admin Only**
+  - name: Namespace Metadata
+    description: APIs to fetch metadata for the namespace. **Admin Only**
+  - name: Cluster State
+    description: APIs to fetch data about the cluster state
+  - name: Datanodes
+    description: APIs to fetch data about the Datanodes
+  - name: Pipelines
+    description: APIs to fetch data about the Pipelines
+  - name: Tasks
+    description: APIs to fetch data about status of Recon Tasks
+  - name: Utilization
+    description: APIs to fetch data about space utilization
+  - name: Metrics
+    description: APIs to fetch data about various metrics from Prometheus
+    externalDocs:
+      description: Prometheus API docs
+      url: https://prometheus.io/docs/prometheus/latest/querying/api/
+paths:
+  /containers:
+    get:
+      tags:
+        - Containers and Keys
+      summary: Get all Container Metadata information
+      operationId: getContainerInfo
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ContainerMetadata'
+  /containers/{containerId}/keys:
+    get:
+      tags:
+        - Containers and Keys
+      summary: Get the Key metadata for all keys present in the given container ID
+      operationId: getContainerKeyMetadata
+      parameters:
+        - name: containerId
+          in: path
+          description: ID of the container for which we want Key Metadata
+          required: true
+          schema:
+            type: integer
+        - name: prevKey
+          in: query
+          description: Only return keys that are present after the given key prevKey prefix
+          required: false
+          schema:
+            type: string
+            example: /vol1/bucket1/key1
+        - name: limit
+          in: query
+          description: Limit of the number of results returned
+          required: false
+          schema:
+            type: integer
+            default: 1000
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/KeyMetadata'
+  /containers/missing:
+    get:
+      tags:
+        - Containers and Keys
```

Comment:
not sure if keys should be included as tag here.

Reply:
So this tag was added as the current recon doc has this specified under Containers and Keys section.
Ref: https://ozone.apache.org/docs/edge/interface/reconapi.html
We can separate it out as Containers only and Keys only

Reply:
> So this tag was added as the current recon doc has this specified under Containers and Keys section. Ref: https://ozone.apache.org/docs/edge/interface/reconapi.html We can separate it out as Containers only and Keys only

Yeah, I meant if API relates to any operation for keys and containers, then having both makes sense, else we can separate out and keep combined one as well.

Reply:
Sure, addressed in the latest commit